frontend/app/src/store/settings/consts.ts

```diff
@@ -1 +1 @@
-export const DEFI_SETUP_DONE = 'defiSetupDone';
+export const DEFI_SETUP_DONE = 'defiSetupDone' as 'defiSetupDone';
```

Comment:
Would it be possible to explain what problem using `as` here solves?

Reply:
As far as I understand, it forces typing `DEFI_SETUP_DONE` as a unique symbol `defiSetupDone` instead of a string. Using that allow me to use it as dynamic props in the component class attribute otherwise I get:
<img width="1340" alt="Screenshot 2020-08-15 at 01 07 47" src="https://user-images.githubusercontent.com/10224453/90298789-cda62d00-de93-11ea-9c3d-da323deea3f7.png">

Reply:
Thank you for the explanation 👍